.github/workflows/debezium-workflow.yml

```diff
@@ -616,6 +616,17 @@ jobs:
           -Dmaven.wagon.http.pool=false 
           -Dmaven.wagon.httpconnectionManager.ttlSeconds=120
 
+      - name: Build Debezium Connector MySQL
```

Comment:
@mfvitale Can this just be solved by changing the name of build core step and adding MySQL to the list of projects?

Reply:
I separated to avoid the `am` flag and avoid to build dependencies on MySQL connector (i.e debezium-storage-file)